src/test/java/org/candlepin/insights/jackson/ObjectMapperContextResolverTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2009 - 2019 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.insights.jackson;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.candlepin.insights.ApplicationProperties;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Calendar;
+import java.util.TimeZone;
+
+
+public class ObjectMapperContextResolverTest {
+
```

Comment:
Mind dropping a comment here?
```suggestion
/**
 * Ensure that dates are in ISO-8601 format
 */
```

Reply:
Comment added.